cmd/authgear/cmd/cmdinternal/e2e.go

```diff
@@ -0,0 +1,45 @@
+package cmdinternal
+
+import (
+	"os"
+
+	authgearcmd "github.com/authgear/authgear-server/cmd/authgear/cmd"
+	"github.com/authgear/authgear-server/cmd/authgear/e2e"
+	"github.com/authgear/authgear-server/pkg/lib/config/configsource"
+	"github.com/authgear/authgear-server/pkg/util/log"
+	"github.com/spf13/cobra"
+)
+
+func NewLoggerFactory() *log.Factory {
+	return log.NewFactory(log.LevelInfo)
+}
+
+var cmdInternalE2EImportUser = &cobra.Command{
+	Use:   "e2e import-users",
+	Short: "Import users for e2e tests",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		binder := authgearcmd.GetBinder()
+
+		configSourceDir := binder.GetString(cmd, authgearcmd.ArgConfigSourceDir)
+		jsonPath := configSourceDir + "/users.json"
```

Comment:
Just read from the first argument. This makes the command easier to use. You can use `Args: cobra.ExactArgs(1)` to validate there is only one argument.